hardhat.config.js

```diff
@@ -34,10 +34,12 @@ module.exports = {
   defaultNetwork: "hardhat",
   networks: {
     hardhat: {
-      // // If you want to do some forking, uncomment this
+      // If you want to do some forking, uncomment this and set `enabled` to true
```

Comment:
I think we can actually leave this uncommented now, and just have `enabled` be `false` to start?